src/pages/chatroom/chatroom.ts

```diff
@@ -140,6 +141,14 @@ export class ChatroomPage {
     image.src = base64String;
   }
 
+  showLoading() {
+    const loader = this.loadingCtrl.create({
+      content: "Uploading...",
+      duration: 3000
```

Comment:
I think we should dismiss the loading controller after sending the message success. Instead of showing it in a fixed duration?